synapse/storage/databases/main/schema/delta/59/1ignored_user.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
can we call this 01ignored_user, to allow space for more than 9 delta files?

Reply:
done